tidy3d/components/grid/corner_finder.py

```diff
@@ -136,5 +221,12 @@ def normalize(v):
         inner_product = np.where(inner_product > 1, 1, inner_product)
         inner_product = np.where(inner_product < -1, -1, inner_product)
         angle = np.arccos(inner_product)
+        num_vs = len(vs_orig)
+        cross_product = np.cross(
```

Comment:
`np.cross` seems to have issues in photonforge: https://github.com/flexcompute/tidy3d/pull/2317

Reply:
should be ok here because that warning is only for 2d vectors, and we have full 3d coordinates here